demo-app/src/lib/mysql-uqi-client.ts

```diff
@@ -136,6 +128,19 @@ async function createMysqlUqiClient(config: MysqlConfig): Promise<UqiClient> {
   })
 }
 
+function azureUserFromUri(parsedUri: URL): string {
```

Comment:
The way this is structured isn't exclusive to azure per se, so I don't think this needs to be named `azureUserFromUri`. But not a big deal either way.

Reply:
The regular expression it uses is azure specific 🤷🏻‍♂️